app/assets/stylesheets/boards.scss

```diff
@@ -0,0 +1,19 @@
+body {
+padding: 10%;
+width: 60em;
+margin: 0 auto 0 auto;
```

Comment:
The `<body>` element effectively represents the entire visible page that all the other elements live on, so it generally shouldn't have margins or a specific width. Padding may make sense in some cases but isn't necessary here since we're using Bootstrap containers, which come with their own padding. Moreover, it would interfere with a top nav, which we'll probably want to add eventually.

Aside from these points, styling the `<body>` in this way should also be avoided for the reason I mentioned earlier in the context of the `.container` class: these CSS rules will apply across the entire site and every page on the site will have a `<body>`.